definitions/2.5.0/parameters.json

```diff
@@ -1,7 +1,11 @@
 {
   "type": "object",
   "additionalProperties": {
-    "$ref": "http://asyncapi.com/definitions/2.5.0/parameter.json"
+    "oneOf": [
+      {
+        "$ref": "http://asyncapi.com/definitions/2.5.0/parameter.json"
+      }
+    ]
```

Comment:
```suggestion
    "oneOf": [
      {
        "$ref": "http://asyncapi.com/definitions/2.5.0/Reference.json"
      },
      {
        "$ref": "http://asyncapi.com/definitions/2.5.0/parameter.json"
      }
    ]
```

Reply:
Made the changes with the commit  [#cdee75e](https://github.com/asyncapi/spec-json-schemas/pull/275/commits/cdee75ef79967df52f1a97c3748581e3a5f62679)